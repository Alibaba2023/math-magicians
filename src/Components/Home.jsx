function Home() {
  return (
    <div className="home-container">
      <p className="home-description">
        &#34;Introducing a sleek and user-friendly web app that combines the
        power of a calculator with the inspiration of quotes. Harnessing
        real-time data from an API, this app effortlessly performs calculations
        while delivering motivational quotes, making number-crunching a
        delightful and uplifting experience. Get ready to calculate and be
        inspired in one place! &#34;
      </p>
    </div>
  );
}
export default Home;
