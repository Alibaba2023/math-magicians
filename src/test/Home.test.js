import React from 'react';
import { render } from '@testing-library/react';
import Home from '../Components/Home';

describe('Home Component', () => {
  test('renders correctly', () => {
    const { container } = render(<Home />);
    expect(container.firstChild).toMatchSnapshot();
  });

  test('has the correct CSS class for home container', () => {
    const { container } = render(<Home />);
    expect(container.firstChild).toHaveClass('home-container');
  });
});
