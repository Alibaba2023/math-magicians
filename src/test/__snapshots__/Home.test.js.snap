// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Component renders correctly 1`] = `
<div
  class="home-container"
>
  <p
    class="home-description"
  >
    "Introducing a sleek and user-friendly web app that combines the power of a calculator with the inspiration of quotes. Harnessing real-time data from an API, this app effortlessly performs calculations while delivering motivational quotes, making number-crunching a delightful and uplifting experience. Get ready to calculate and be inspired in one place! "
  </p>
</div>
`;
